adabot/circuitpython_bundle.py

```diff
@@ -419,6 +420,26 @@ def add_contributors(master_list, additions):
         master_list[contributor] += additions[contributor]
 
 
+def test_bundle_build(bundle_dir):
+    """
+    Attempts to build the bundle at the given location.
+    Returns exit code 0 if success.
+    Returns exit code >0 if failed to build.
```

Comment:
I don't see a return value.

```suggestion
```

Reply:
@tannewt I'm not certain of the proper terminology, "Returns" is probably not correct. Maybe "Raises"? Basically these comments are talking about the sys.exit(2) from here: https://github.com/adafruit/circuitpython-build-tools/blob/1f3c5bd0a250dc162306a522c3ee5cd2c117716b/circuitpython_build_tools/scripts/build_bundles.py#L208

test_build_bundle function will allow that SystemExit (2) value to bubble upward which then gets caught down at 589. 


Alternatively I could move the try / catch SystemExit(2)  to inside of test_build_bundle() and then actually make it return the integer value so that the current documentation wording will be accurate.

Reply:
"Raises SystemExit with a nonzero value if failed to build"?

Reply:
re-worded with the latest commit. Thank you @jepler!